docs/generating_asts.rst

```diff
@@ -134,7 +134,56 @@ Parameters used by the supplementing AST method
 - `ast_suppl_maglimit` : dictionary (optional)
 
   If supplied, these magnitude limits will be applied to the SED model grids
-  when selecting additional ASTs.
+  when selecting additional ASTs. This is a dictionary that includes information
+  for the magnitude cuts as a function of the filters included in observation.
+
+  For example, for a field observed with HST_WFC3_F475W and HST_WFC3_F814W,
+  to set a magnitude range limit of 16<HST_WFC3_F475W<28 mag,
+  and 15<HST_WFC3_F814W<27 mag you need to set the following within the beast_settings file:
+
+  .. code-block:: console
+
+    # initialize and populate the dictionary of desired magnitude limits
+    ast_suppl_maglimits = {}
+
+    # the magntidue limits are defined by the filter and a list of the limits in magnitudes
+    ast_suppl_maglimits["HST_WFC3_F475W"] = [16,28]
+    ast_suppl_maglimits["HST_WFC3_F814W"] = [15,27]
+
+    # set the key word
+    ast_suppl_maglimit = ast_suppl_maglimits
+
+  or, equivalently:
+
+  .. code-block:: console
+
+      ast_suppl_maglimit = {‘F475W’: (16,28), ‘F814W’: (15,27)}
+
+
+- `ast_suppl_colorlimit` : dictionary (optional)
+
+  If supplied, these color limits will be applied to the SED model grids
+  when selecting additional ASTs. This is a dictionary that includes information
+  for the color cuts as a function of the filters included in observation.
+
+  For example, for a field observed with HST_WFC3_F336W, HST_WFC3_F475W,
+  and HST_WFC3_F814W, to set a color range limit of HST_WFC3_F475W-HST_WFC3_F814W<6,
+  HST_WFC3_F336W-HST_WFC3_F475W<5 and HST_WFC3_F336W-HST_WFC3_F814W<4, you need
+  to set the following within the beast_settings file:
+
+  .. code-block:: console
+
+    # initialize the dictionary of desired magnitude limits
+    ast_suppl_colorlimits = {}
+
+    # the color limits are defined by the first filter in the color (e.g, X for X-Y),
+    # and the input is a list including the second filter (e.g., Y for X-Y) and the
+    # color limit in magnitudes:
+    ast_suppl_colorlimits["HST_WFC3_F475W"] = [["HST_WFC3_F814W",4]]
```

Comment:
Same comment for the beast/observationmodel/ast/make_ast_input_list.py.
ast_suppl_colorlimits["HST_WFC3_F475W"] = [["HST_WFC3_F814W",6]]
ast_suppl_colorlimits["HST_WFC3_F336W"] = [["HST_WFC3_F475W",5], ["HST_WFC3_F814W",4]]